samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<p>Seems the word cameras might be a typo in this sentence:   &nbsp;<em>Locate a place, drag the map around, and then run the last cell to plot a heat map of recycling amenities cameras and all of the shops within the extent of the map.</em>&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>